src/planscape/collaboration/views.py

```diff
@@ -46,3 +50,20 @@ def post(self, request, format=None):
             logger.exception("Something failed during the creation of a new invite.")
             # this will return a 500
             raise
+
+
+class UserObjectRolesForObject(APIView):
+
+    def get(self, request: Request, target_entity: str, object_pk: int):
+
+        user = request.user
+        if not check_for_permission(user.pk, "add_collaborator"):
+            return Response(status=status.HTTP_403_FORBIDDEN)
+
+        content_type = ContentType.objects.get(model=target_entity)
+        user_object_roles = UserObjectRole.objects.filter(
+            content_type=content_type,
+            object_pk=object_pk,
+        ).exclude(collaborator_id=user.pk)
```

Comment:
not sure what this means, we are excluding the user_object_roles which you are a collaborator? Seem like we want the oposite